README.md

```diff
@@ -87,6 +87,21 @@ f5_vip 'myvip' do
   #                   anything else is a named firewall_policy
   staged_firewall_policy
 end
+
+# Create a F5 monitor template
+f5_monitor 'test-monitor' do 
+  # Required properties
+  template_type 'TTYPE_HTTP' # F5 template type
+  parent_template 'http'
+  interval 5 # frequency to check in seconds
+  timeout 3 # seconds to respond
+  
+  #optional properties (default values are shown)
+  dest_ip '0.0.0.0' # tells f5 what address to test. Default indicates to test each node in pool 
```

Comment:
The wording seems odd here. I know there's some translation between the API and the web UI, but I _think_ this should say something more like `default indicates to test the node the monitor is applied to`. You can tell the F5 to check a different host (or port) than the one this monitor template is attached to.